lib/ddtrace/contrib/active_record/patcher.rb

```diff
@@ -1,11 +1,24 @@
+require 'ddtrace/contrib/rails/utils'
+require 'ddtrace/ext/ruby'
+require 'ddtrace/ext/sql'
+require 'ddtrace/ext/app_types'
+
 module Datadog
   module Contrib
     module ActiveRecord
       # Patcher enables patching of 'active_record' module.
       module Patcher
         include Base
         register_as :active_record, auto_patch: false
-        option :service_name
+        option :service_name, default: 'active_record', depends_on: [:tracer] do |value|
```

Comment:
Also not sure the AR part should be called `active_record` by default. Suppose you create a parent span with name `my-app`, probably you want to have it `my-app` instead of `active_record` (so you don't create another service in your list but you only have `my-app` that includes the AR latency.

This should handle only the case where you use a standalone AR without a parent span, but in that case maybe we should introduce more the concept of default service for a Tracer.

Reply:
I think we can change `service_name`  to configure both the `database` and `service` parts, but we will not be able to introduce independently configurable service names without introducing a breaking change at some point, since ultimately `service_name` is actually a database name right now.

We'd need to ask users to change their configuration: something we could ask via deprecation warning perhaps?